safehttp/plugins/fetch_metadata/fetch_metadata.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fetchmetadata
+
+import (
+	"github.com/google/go-safeweb/safehttp"
+)
+
+// RequestLogger is a user-provided service for logging Fetch Metadata policy
+// violations
```

Comment:
nit: missing dot at the and, and it's enough to say: `RequestLogger is used for logging Fetch Metadata policy violations`. "user-provided" doesn't add any meaning here.